docs/runner/events.create.rst

```diff
@@ -0,0 +1,27 @@
+.. _events_create_yamlref:
+
+Events Create Story Runner YAML
+...........................................
+
+'asset_name' is required from a previously created asset. The asset_id is derived and 
+inserted as the first argument to events.create().
+
+'args' is required - the first value is the 
+props argument and the second value is the attrs argument to events.create().
+
+.. code-block:: yaml
+    
+    ---
+    operations:
+      - operation: events.create
+        to_print: Create Event requesting EV pump 1 needs maintenance.
+        asset_name: ev pump 1
+        args:
+          - operation: Record
+            behaviour: RecordEvidence
+          - arc_correlation_value: EV Maintenance 1
+            arc_description: request maintenance
+            arc_evidence: see attached report
```

Comment:
Again probably better to have a self-explanatory string here (like 'Service cycles exceeded') if we're not actually adding attachments to the Event